en/docs/deploy/fronting-with-the-nginx-load-balancer.md

```diff
@@ -0,0 +1,257 @@
+# Set up WSO2 clusters with Nginx 
+
+When setting up the WSO2 Identity Server cluster with Nginx, follow the instructions given below. 
+
+!!! note 
+    Follow the instructions given below only **after** setting up the cluster following the [Deployment Patterns](../../../deploy/deployment-guide). 
+
+!!! tip 
+    When clustering WSO2 Identity Server with a load balancer, you may need to
+    enable sticky sessions. This is required for the management console and
+    the user portal to work and if we disable temporary session data
+    persistence in the
+    `<IS_HOME>/repository/conf/deployment.toml`
+    file. For more information on sticky sessions, see [Sticky Sessions with Manager Nodes](../../../deploy/sticky-sessions-with-manager-nodes).
+   
+Following is the deployment diagram with the load balancer.
+
+![Deployment with the loadbalancer](../../assets/img/deploy/deployment-with-the-loadbalancer.png) 
```

Comment:
does this need three dots?

Reply:
Works with two dots for me. I guess it's because it does not have a sub folder within "deploy".